print/resources/inkstitch.js

```diff
@@ -303,6 +368,7 @@ $(function() {
     var field_name = $(this).attr('data-field-name');
 
     $('.' + field_name).toggle($(this).prop('checked'));
+    scaleAllSvg();
```

Comment:
@kaalleen I think this fixes a problem I saw where enabling the client detailed view shows empty boxes until you do "apply to all", is that right?

Reply:
Yes, that's what I wanted to do there ;)